### PR TITLE
OUD 2016: list of accepted presentations

### PR DESCRIPTION
**Pull request by gasche**

### CODE REVIEW

File: site/meetings/ocaml/2016/index.md [hunk 1]
```diff
@@ -25,6 +19,65 @@ Previous editions have been colocated with ICFP 2012 in
 ICFP 2015 in [Vancouver](../2015),
 following the OCaml Meetings that ran until [2011](../2011).
 
+Presentations (past)
```
Comment (amirmc) [inline site/meetings/ocaml/2016/index.md]: @gasche Did you mean to say "(past)"?

Comment (gasche) [inline site/meetings/ocaml/2016/index.md]: I meant to qualify "Call for Papers" as "(past)", sorry. I think that when the definitive programme is available I will similarly push the list of accepted presentations to the bottom -- I don't remember how to create link anchors with that markdown syntax, that might be puseful. I fixed the PR.

File: site/meetings/ocaml/2016/index.md [hunk 2]
```diff
@@ -25,6 +19,65 @@ Previous editions have been colocated with ICFP 2012 in
...
+====================
+
+OCaml 2016 will open with an invited talk by Damien Doligez.
+
+The following works will be presented as a 20 minute talks.
```
Comment (mshinwell) [inline site/meetings/ocaml/2016/index.md]: "as" not "as a"

**Comment by mshinwell** [comment]
The Travis failure appears unrelated, so I'm merging this.

**Pull request merged**